charts/k8tz/templates/certificate.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if .Values.webhook.certManager.enabled -}}
+{{- $fqdn := printf "%s.%s.svc" (include "k8tz.serviceName" .) .Values.namespace }}
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: {{ include "k8tz.fullname" . }}-tls
+  namespace: {{ .Values.namespace }}
+  labels:
+    {{- include "k8tz.labels" . | nindent 4 }}
+spec:
```

Comment:
what about `secretTemplate`,  `duration` and `renewBefore`, do you think we should support them as well?

Reply:
yes, we can support the `secretTemplate` value. but I don't think so with the `duration` and `renewBefore`.
since by default it has 90days of duration, and auto renew on 2/3 of total duration days.